entity/src/advisory.rs

```diff
@@ -9,6 +10,10 @@ pub struct Model {
     pub location: String,
     pub sha256: String,
     pub title: Option<String>,
+    pub aggregate_severity: Option<String>,
```

Comment:
Short term I think that's ok. Long term we must also have the namespace alongside that. "medium" from Red Hat might mean something different than "medium" from Acme Corp.

Reply:
I see. Just a further question just to understand or open a discussion:

`severity` can mean something different for VendorA and for VendorB. Does it mean that:
- Option1: regardless of how VendorA and VendorB see `severity` we are going to respect what is written is the file itself and therefore render in the UI (or expose through the api) the exact `severity` defined in the file?
  - If we will respect what is written in the file perhaps the `namespace` does not add any value? Perhaps a field `vendor` is what would make sense to clearly express that the file belongs to certain vendor?
- Option2: because "medium" for Red Hat might might mean "critical" from Acme Corp (I changed your example to make the case). Does it mean we are going to come up with our own system for `severity` so all vendors are aligned to our system of categorizing severities?

I could've just left the namespace for "later" but I wanted to ask just for my understanding. If I'm not making any sense just ignore my message xD